### PR TITLE
Follow-up to PR 10566

### PR DESCRIPTION
**Pull request by ghost**
@bfallonf Can you peer review this pull request? It's a follow-up to https://github.com/openshift/openshift-docs/pull/10566. 

@vikram-redhat I have no idea why the Travis check fails because all of those IDs are pointing to existing files/sections.

### CODE REVIEW

File: install_config/persistent_storage/topics/glusterfs_uninstall.adoc [hunk 1]
```diff
@@ -8,15 +8,18 @@ ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansibl
 ----
```
Comment (bfallonf) [inline install_config/persistent_storage/topics/glusterfs_uninstall.adoc]: @tmorriso-rh I failed to notice this in the original PR, but the above command needs a prompt before it. So, `#`, I'd say?

File: install_config/persistent_storage/topics/glusterfs_uninstall.adoc [hunk 2]
```diff
@@ -8,15 +8,18 @@ ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansibl
...
+block devices that were used for {gluster} backend storage. To use the
+`openshift_storage_glusterfs_wipe` variable:
 
 ----
-ansible-playbook -i <path_to_inventory_file> -e "openshift_storage_glusterfs_wipe=true" /usr/share/ansible/openshift-ansible/playbooks/openshift-glusterfs/uninstall.yml
+---- ansible-playbook -i <path_to_inventory_file> -e
 ----
+"openshift_storage_glusterfs_wipe=true"
+----
+/usr/share/ansible/openshift-ansible/playbooks/openshift-glusterfs/uninstall.yml ----
```
Comment (bfallonf) [inline install_config/persistent_storage/topics/glusterfs_uninstall.adoc]: @tmorriso-rh I'm not sure what's happening above. There's been some extra ----'s added into the middle of the command? Looks like the command hasn't changed, so maybe just delete the extra lines that have been added.

You can see it better here:

https://raw.githubusercontent.com/tmorriso-rh/openshift-docs/0ffd0d4350d5a39e28fd366025db447feb7fb82f/install_config/persistent_storage/topics/glusterfs_uninstall.adoc

Also, like the other comment I made above, this command needs a prompt at the start.

**Comment by vikram-redhat** [comment]
@tmorriso-rh - Yes, as @bfallonf said, it seems that the command has been broken down for some reason with some ----'s added here: https://github.com/openshift/openshift-docs/pull/10744/files#diff-e7f6eeff45272148f4f08a56d632c6e8R16, which is quite likely causing the book from being built properly.

**Pull request merged**